Endava.TechCourse.BankApp.Application/Commands/AddWalletType/AddWalletTypeHandler.cs

```diff
@@ -0,0 +1,36 @@
+using Endava.TechCourse.BankApp.Application.Commands.AddCurrency;
+using Endava.TechCourse.BankApp.Domain.Models;
+using Endava.TechCourse.BankApp.Infrastructure.Persistence;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Endava.TechCourse.BankApp.Application.Commands.AddWalletType;
+
+public class AddWalletTypeHandler : IRequestHandler<AddWalletTypeCommand, CommandStatus>
+{
+    private readonly ApplicationDbContext context;
```

Comment:
When you are using private proprieties, their are 2 conventions that you can apply :
1) their name must start with "_"  =>  private readonly ApplicationDbContext _context
2)use "this." keyword => this.context = context and so on